cron/monthly/verify-plans.js

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+/**
+ * Makes sure all collectives.plan are up-to-date, downgrading and cancelling plans if needed.
+ */
+
+// Only run on the first of the month
+const today = new Date();
+if (process.env.NODE_ENV === 'production' && today.getDate() !== 1) {
+  console.log('NODE_ENV is production and today is not the first of month, script aborted!');
+  process.exit();
+}
+
+process.env.PORT = 3066;
```

Comment:
What's that?

Reply:
Not sure, it is included in every monthly cron worker, I thought it was a dependency related hack.